addons/mass_mailing/static/src/js/mass_mailing.js

```diff
@@ -38,7 +38,7 @@ FieldTextHtml.include({
         var datarecord = this._super();
         if (this.model === 'mail.mass_mailing') {
             // these fields can potentially get very long, let's remove them
-            datarecord = _.omit(datarecord, ['mailing_domain', 'contact_list_ids', 'body_html']);
+            datarecord = _.omit(datarecord, ['mailing_domain', 'contact_list_ids', 'body_html', 'attachment_ids']);
```

Comment:
The fix seems right for stable. Now, @Gorash, why do we need to pass the whole daterecord by URL instead of its id only?

Reply:
@qsm-odoo As it's a fix, we have to be careful of potential customizations.
But you are right, in master we should use id only.